attributes/default.rb

```diff
@@ -23,3 +23,4 @@
 default['composer']['global_configs'] = {}
 default['composer']['home_dir'] = nil
 default['composer']['php_recipe'] = 'php::default'
+default['composer']['selfupdate_channel'] = nil
```

Comment:
Nitpicking-mode: any chance of making this:
`default['composer']['self_update']['channel'] = nil`